### DISCUSSION TITLE
Is zooming implemented on desktop? How to zoom with keyboard or mouse？

### DISCUSSION THREAD

**Discussion by wangzhentian** [Live-Charts/LiveCharts2#673] [Q&A]
Newbee question, thanks!

**Comment by beto-rodriguez** [comment]
Hello! 

To zoom in/out, use the mouse wheel, you must also enable it using the `ZoomMode` property.

For more info please see https://lvcharts.com/docs/WPF/2.0.0-beta.330/samples.lines.zoom

**Comment by wangzhentian** [comment]
Hello Alberto:

Thanks for the info. Must be something wrong with my OS or hardware. The zoom doesn't work for me even when I run the Zoom demo. I will test on other machinese and provide more feedbacks.

**Reply by pavlexander** [reply]
Same. Zoom does not work in WinForms for me..
